providers/databricks/src/airflow/providers/databricks/utils/databricks.py

```diff
@@ -64,3 +64,22 @@ def validate_trigger_event(event: dict):
         RunState.from_json(event["run_state"])
     except Exception:
         raise AirflowException(f'Run state returned by the Trigger is incorrect: {event["run_state"]}')
+
+
+def validate_serverless_notebook_settings(content) -> bool:
```

Comment:
nit: it is better to not return anything since we failing validation by throwing exception
```suggestion
def validate_serverless_notebook_settings(content):
```